### PR TITLE
adds Expander Page

### LINKED ISSUE #153: Add page for Expander component
Add a new page for the [Expander component](https://github.com/microsoft/fluentui-react-native/pull/756) recently added to FURN. Here is the [README for the Expander](https://github.com/amdingler/fluentui-react-native/blob/windows-native-expander/packages/experimental/Expander/README.md).

Note: The app will also need to be updated to WinUI 2.6

### PR DESCRIPTION
**Pull request by TatianaKapos**
Adds an [Expander](https://github.com/microsoft/fluentui-react-native/blob/master/packages/experimental/Expander/README.md) Page to Gallery App

snapshot of page
image

added Example with interactive controls
image

closes #153

### CODE REVIEW

File: src/examples/ExpanderExamplePage.tsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+      componentType="Community"
+      pageCodeUrl="https://github.com/microsoft/react-native-gallery/blob/main/src/examples/ExperimentalExpanderExamplePage.tsx"
+      documentation={[
+        {
+          label: 'Expander',
+          url:
+            'https://github.com/microsoft/fluentui-react-native/blob/master/packages/experimental/Expander',
+        },
+      ]}>
+      <Example title="A simple Expander" code={example1jsx}>
```
Comment (chiaramooney) [inline src/examples/ExpanderExamplePage.tsx:59]: nit: RN Gallery titles match XAML Control Gallery title syntax where each title ends with a period.

PS: "nit" before a comment on a PR means nitpicky.

File: src/examples/ExpanderExamplePage.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+          collapsedHeight={50}
+          expandedHeight={100}
+          headerBackground={colors.border}
+          headerForegroundPointerOver={colors.primary}
+          contentBackground={colors.primary}
+          contentBorderBrush={colors.background}>
+          <Text>Text in the header</Text>
+          <Text>Text in the content</Text>
+        </Expander>
+      </Example>
```
Comment (chiaramooney) [inline src/examples/ExpanderExamplePage.tsx:97]: Could you add one more example that showcases having interactive controls inside of the Expander header and content such as a checkbox or datetimepicker?

File: src/examples/ExpanderExamplePage.tsx [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+  contentBackground="#00adee"
+  contentBorderBrush="#fabc09">
+    <Text>Text in the header</Text>
+    <Text>Text in the content</Text>
+</Expander>`;
+
+  return (
+    <Page
+      title="Expander"
+      description="Expander is a control that displays a header and has a collapsible body that displays content"
```
Comment (chiaramooney) [inline src/examples/ExpanderExamplePage.tsx:44]: nit: Add period to end of description.

File: src/examples/ExpanderExamplePage.tsx (hunk 2 above)
Comment (TatianaKapos) [reply src/examples/ExpanderExamplePage.tsx:111]: added an example with checkbox in header and datetimepicker in content.

**Review by chiaramooney — APPROVED** [review]
Looks great!

File: src/examples/ExpanderExamplePage.tsx [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+          expandedHeight={100}
+          headerBackground={colors.border}
+          headerForegroundPointerOver={colors.primary}
+          contentBackground={colors.primary}
+          contentBorderBrush={colors.background}>
+          <Text>Text in the header</Text>
+          <Text>Text in the content</Text>
+        </Expander>
+      </Example>
+      <Example title="A Expander with interactive controls." code={example5jsx}>
```
Comment (chrisglein) [inline src/examples/ExpanderExamplePage.tsx:112]:
```suggestion
      <Example title="An Expander with interactive controls." code={example5jsx}>
```

File: src/examples/ExpanderExamplePage.tsx [hunk 5]
```diff
@@ -0,0 +1,132 @@
...
+            <CheckBox onCheckColor={colors.primary} />
+            <Text
+              style={{
+                alignItems: 'center',
+                padding: 6,
+              }}>
+              Click Me!
+            </Text>
+          </View>
+          <DateTimePicker value={date1} onChange={() => {}} mode="date" />
```
Comment (chrisglein) [inline src/examples/ExpanderExamplePage.tsx:127]: Why the empty `onChange` handler?

**Review by chrisglein — APPROVED** [review]

File: src/examples/ExpanderExamplePage.tsx (hunk 5 above)
Comment (TatianaKapos) [reply src/examples/ExpanderExamplePage.tsx]: copied and pasted the component from DateTimePicker page so I thought it was needed for the component. The onChange handler is now taken out! :)

**Pull request merged**